### ISSUE TITLE
Error running 'Digitis Fracture Network'

### ISSUE THREAD

**Issue by campbellOR** [BjornNyberg/NetworkGT#21]
I have been attempting to digitise fracture networks from fracture maps thresholded using the NetworkGT plugin. However, when I run the digitise fracture network tool, I receive this error:
![image](https://user-images.githubusercontent.com/47636928/177006711-fb4fe1e8-20a6-464c-9baa-ac6890d7f4eb.png)
Looking in my temp files, I have a .tiff file of the same name, but not the .shp that it says it cannot find.
Any help in solving this issue would be greatly appreciated.
Cheers

**Comment by BjornNyberg** [comment]
Hi @campbellOR 

This is the first time that I have seen this type of error. Do you have an example to share to see if one can replicate it? I also would try and save the files rathet than using temporary memory files to see if that resolves it.

Otherwise we can always have a quick zoom chat to resolve the issue.

Cheers
Bjorn

**Comment by campbellOR** [comment]
Hi @BjornNyberg 
I had the same issue come up even while trying to save the layer instead of creating a temp layer. On some examples unchecking the 'Open output file after running algorithm' solved it, but others it didn't. I eventually managed to get it to work by changing the source layer CRS, attempting the algorithm, thne reverting to the correct CRS and trying again. So I have solved the issue for the dataset I am working on, and I can't seem to recreate the error message in an example to send.
Cheers,
Ollie
